scania-digital-design-system/tegel
# fix(storybook-colors): radio controls over dropdown

**Describe pull-request**  
Scale color story to use radio buttons instead of dropdown

**Solving issue**  
Fixes: DTS-1716

**How to test**  
1. Go to the Storybook link below
2. Check "Colors" -> "Scale" story
3. Control for switching value is a radio button and not a dropdown as in the main branch.

**Note**
Readme files of other components are also updated in this PR too. Suggestion it so look just to commit for this task.
